wgsl/index.bs

```diff
@@ -4127,7 +4134,7 @@ specify the component type; the component type is inferred from the constructor
 </table>
 
 <table class='data'>
-  <caption>Vector constructor type rules, where *T* is a scalar type</caption>
+  <caption>Vector constructor type rules</caption>
```

Comment:
The type T is already implicitly constrained by the availability of types in the conclusion.

This is way of constraining is already used in the matrix-construction tables.

So LGTM.